integration-tests/src/test/groovy/com/okta/sdk/tests/it/EventHooksIT.groovy

```diff
@@ -15,13 +15,12 @@
  */
 package com.okta.sdk.tests.it
 
-import com.okta.sdk.impl.resource.EventHookBuilder
+import com.okta.sdk.impl.resource.DefaultEventHookBuilder
```

Comment:
Looks like this is missing a tag, I'm guessing it should be `Hooks`

Reply:
OH sorry, that is the builder, so two things 
1.) There should not be an `instance()` method on `DefaultEventHookBuilder` it should be `com.okta.sdk.resource.hooks.EventHookBuilder.instance()` (and from the test you could call either `new DefaultEventHookBuilder()` or the factory method)
2.) It's missing a package declaration

Reply:
sorry, i guess it should be something like:
`com.okta.sdk.resource.event.hook.EventHookBuilder.instance()`

Reply:
The builder interfaces are always flat (or as flat as we can make them and avoid the extra nesting of the resource classes under the covers.


Something like (removing the static values, which the builder should default to):
```java
EventHook createdEventHook = EventHookBuilder.instance()
            .setName(name)
            .setUrl("https://www.example.com/eventHooks")
            .setAuthorizationHeaderValue("Test-Api-Key")
            .addHeader("X-Test-Header", "Test header value"))
            .buildAndCreate(client);
```

Reply:
Updated.